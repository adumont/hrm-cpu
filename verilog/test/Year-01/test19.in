52 8a
05 b4
16 b2
57 78
29 c4
32 a9
22 eb
