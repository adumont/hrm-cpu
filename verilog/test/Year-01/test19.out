8a
b4
b2
